src/view/viewTabs.component.tsx

```diff
@@ -21,6 +22,7 @@ export const paths = {
   manufacturer: '/manufacturer/:id',
   catalogueItem: '/catalogue/item/:id',
   items: '/catalogue/item/:id/items',
+  item: '/catalogue/item/:id/items/:id',
```

Comment:
Couldn't this cause confusion for `useParams`? - it seems like it is using the last id which is fine, but should probably have something like
`/catalogue/item/:catalogue_item_id/items/:item_id`
to be more specific. The other are fine as they are the only one although if it is changed here it might be worth making them consistent.